AzureStore.UI/app/src/components/OrderForm/OrderForm.tsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+
+const emptyOrder = {
+    userEmail: "",
+    orderItems: []
+};
+
+interface Props {
+    submitOrder: (order: any) => void;
+    selectedItems: Array<any>;
+}
+
+interface State {
+    userEmail: string,
+    orderItems: []
+}
+
+export default class OrderForm extends React.Component<Props, State> {
+
+    constructor(props: any) {
```

Comment:
no need in ctor;
state could be initialized separately 
`    state = {
        maxPrice: 0,
        page: 1
    };`